media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,984 @@
+        }
+        span {
+            .visually-hidden();
+        }
+    }
+    h2 {
+        .open-sans-light();
+        .font-size(3rem);
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(2.2rem);
+            line-height: 1.1;
+        }
+    }
+    h2, h3 {
+        color: white;
+        text-shadow: none;
+    }
+    .tagline {
+        .open-sans-light();
+        color: #bee1f5;
+        margin-bottom: 20px;
+        font-size: 2em;
+        letter-spacing: -0.03em;
+
+        @media only screen and (max-width: 1366px) {
+            line-height: 1.1;
+        }
+        @media only screen and (max-width: 1024px) {
+            margin-bottom: 15px;
+            .font-size(1.6rem);
+            line-height: 1.3;
+        }
+        @media only screen and (max-width: @breakMobileLandscape) {
+            margin-bottom: 20px;
+            .font-size(1.4rem);
+        }
+    }
+}
+.content-wrapper {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1400px;
+}
+/* Buttons */
+.cta-button {
+    display: inline-block;
+    background-color: #005189;
+    margin: 0 auto 30px;
+    border: 0;
+    border-radius: 6px;
+    color: white;
+    font-size: 1.4em;
+    padding: .7em 3em;
+
+    @media only screen and (max-width: 1600px) {
+        margin-bottom: 50px;
+    }
+
+    @media only screen and (max-width: 1366px) {
+        margin-bottom: 30px;
+    }
+
+    @media only screen and (max-width: 1024px) {
+        margin-bottom: 20px;
+    }
+
+    &:hover,
+    &:focus,
+    &:active {
+        background-color: #005da5;
+        color: white;
+        text-decoration: none;
+    }
+}
+.get-phone,
+.get-newsletter {
+    display: block;
+    width: 13em;
+    min-width: 13em;
+    &.hidden {
+        display: none;
+    }
+}
+.get-newsletter {
+    margin-bottom: 15px;
+}
+.fxos-hero-space {
+    color: white;
+    padding-top: 3em;
+    height: 415px;
+    overflow: hidden;
+    clear: both;
+    .transition(height .3s);
+
+    @media only screen and (max-width: 1600px) {
+        padding-top: 2em;
+        height: 415px;
+    }
+    @media only screen and (max-width: 1366px) {
+        height: 375px;
+    }
+    @media only screen and (max-width: 1024px) {
+        height: 295px;
+        padding-top: 1.5em;
+    }
+    @media only screen and (max-width: @breakTablet) {
+        height: 595px
+    }
+    @media only screen and (max-width: @breakMobile) {
+        padding-top: 1em;
+        height: 550px;
+    }
+
+    ul {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+    }
+}
+/* class is added to fxos-hero-space when switching to demo mode */
+.trim-height {
+    height: 900px;
+}
+.phone {
+    position: relative;
+    top: 0;
+    margin: 0 auto;
+    opacity: 1;
+    .transition(all .7s);
+    .signup {
+        margin-bottom: 25px;
+    }
+    .signup-txt {
+        color: #BEE1F5;
+        margin: 0 auto;
+        width: 18em;
+        .font-size(1.4rem);
```

Comment:
Here's the culprit on the font size, this is calculated to 22px. Presumably this should be <code>.font-size(12px);</code>?